src/commands/roadmap.ts

```diff
@@ -0,0 +1,12 @@
+import { CommandDefinition } from '../lib/command';
+import { makeEmbed } from '../lib/embed';
+import { CommandCategory } from '../constants';
+
+const ROADMAP_URL = 'https://cdn.discordapp.com/attachments/740722295009706034/893674991667798026/ROADMAP.png';
+
+export const roadmap: CommandDefinition = {
+    name: 'roadmap',
```

Comment:
```suggestion
    name: ['roadmap', 'goals'],
```

Could add a goals alias?